main.py

```diff
@@ -59,7 +59,9 @@ def convert():
     try:
       return perform_notebook_conversion(json)
     except Exception as e:
-      return render_template('jupyter-error.html', error=e.__class__.__name__ + ": " + str(e) + ". ") ,400
+      errMessage = f'{e.__class__.__name__} : {''.join(str(e).splitlines())} .'
```

Comment:
I think using the single quotes here in the .join are messing with the f string

Reply:
I don't know what you're talking about 👀 😜